nbdev/frontmatter.py

```diff
@@ -12,13 +12,17 @@
 import yaml
 
 # %% ../nbs/API/frontmatter.ipynb 5
-_re_fm = re.compile(r'''^---\s*
-(.*\S+.*)
----\s*$''', flags=re.DOTALL)
+_RE_FM_BASE=r'''^---\s*
+(.*?\S+.*?)
```

Comment:
You don't need these `?` since `*` can be zero chars

Reply:
resolved on skype